test/parsemultipart.jl

```diff
@@ -3,7 +3,31 @@ using HTTP
 import HTTP.MultiPartParsing: find_multipart_boundary, find_multipart_boundaries, find_header_boundary, parse_multipart_chunk, parse_multipart_body, parse_multipart_form
 
 function generate_test_body()
-    Vector{UInt8}("----------------------------918073721150061572809433\r\nContent-Disposition: form-data; name=\"namevalue\"; filename=\"multipart.txt\"\r\nContent-Type: text/plain\r\n\r\nnot much to say\n\r\n----------------------------918073721150061572809433\r\nContent-Disposition: form-data; name=\"key1\"\r\n\r\n1\r\n----------------------------918073721150061572809433\r\nContent-Disposition: form-data; name=\"key2\"\r\n\r\nkey the second\r\n----------------------------918073721150061572809433\r\nContent-Disposition: form-data; name=\"namevalue2\"; filename=\"multipart-leading-newline.txt\"\r\nContent-Type: text/plain\r\n\r\n\nfile with leading newline\n\r\n----------------------------918073721150061572809433--\r\n")
+    Vector{UInt8}("""----------------------------918073721150061572809433\r
```

Comment:
Looks scary to rely on `\n`s here -- does string literals use `\n` regardless of the line endings of the file?
Perhaps write this as
```
Vector{UInt8}(join([
    "line1",
    "line2",
], "\r\n"))
```
or something?

Reply:
Great point! Love that suggestion. Thanks 👍
Will change this when I'm back to the computer later.

Reply:
Done.